commitlint/plugins.test.ts

```diff
@@ -778,6 +778,30 @@ test("footer-refs-validity8", () => {
     expect(footerRefsValidity8.status).toBe(0);
 });
 
+// This test reflects this issue: https://github.com/nblockchain/conventions/issues/148
+test("footer-refs-validity9", () => {
+    let commitMsgWithCodeBlockAtBodyAndRef =
+        "foo: blah blah" +
+        "\n\n" +
+        "Blah blah blah [2]." +
+        "\n\n" +
+        "Blah blah:" +
+        "\n\n" +
+        "```\nsomeCodeBlock\n```" +
+        "\n\n" +
+        "Blah blah blah: [1]." +
+        "\n\n" +
+        "[1] someUrl://blahblah.com" +
+        "\n" +
+        "[2] Stack trace:" +
+        "\n\n" +
+        "```\nsomeCodeBlock\n```";
```

Comment:
@Mersho now that you understand the issue better (because you have found a fix), cannot you simplify this testcase? I mean, the first thing I see that could be simplified is that the latter code block is not needed to reproduce the bug, right?

Reply:
yes, you're right.